examples/README.md

```diff
@@ -10,40 +10,41 @@ Notably, `capture` example shows rendering without a surface/window. It reads ba
-| off-screen rendering   |        |        |        |           | :star: |        |                | :star: |
-| stencil testing        |        |        |        |           |        |        |                |        |
-| depth testing          |        |        |        |           | :star: | :star: |                | :star: |
-| depth biasing          |        |        |        |           | :star: |        |                |        |
-| read-only depth        |        |        |        |           |        |        |                | :star: |
-| blending               |        | :star: |        |           |        |        |                | :star: |
-| render bundles         |        |        |        | :star:    |        |        |                | :star: |
-| compute passes         | :star: |        |        |           |        |        |                |        |
-| *optional extensions*  |        |        |        |           |        |        | :star:         |        |
-| - binding indexing     |        |        |        |           |        |        | :star:         |        |
-| - push constants       |        |        |        |           |        |        | :star:         |        |
-| - depth clamping       |        |        |        |           | :star: |        |                |        |
-| - compressed textures  |        |        |        |           |        | :star: |                |        |
-| - polygon mode         |        | :star: |        |           |        |        |                |        |
-| - queries              |        |        | :star: |           |        |        |                |        |
-| *integrations*         |        |        |        |           |        |        |                |        |
-| - staging belt         |        |        |        |           |        |        |                |        |
-| - typed arena          |        |        |        |           |        |        |                |        |
-| - obj loading          |        |        |        |           |        | :star: |                |        |
+| Feature                      | boids  | cube   | mipmap | msaa-line | shadow | skybox | texture-arrays | water  | conservative-raster |
+| ---------------------------- | ------ | ------ | ------ | --------- | ------ | ------ | -------------- | ------ | ------------------- |
+| WGSL shaders                 | :star: | :star: | :star: | :star:    | :star: | :star: |                |        | :star:              |
+| vertex attributes            | :star: | :star: |        | :star:    | :star: | :star: | :star:         | :star: | :star:              |
+| instancing                   | :star: |        |        |           |        |        |                |        |                     |
+| lines and points             |        |        |        | :star:    |        |        |                |        | :star:              |
```

Comment:
conservative doesn't apply to lines and points, so wonder why using them in the example?

Reply:
I'm using it to draw "the real triangle" over the pixelated one
https://github.com/Wumpf/wgpu-rs/blob/conservative-raster/examples/conservative-raster/screenshot.png
I'd really like to keep that because it's a nice visualization. But with a bit of effort I can make it optional